docs/source/config_file.rst

```diff
@@ -377,8 +377,9 @@ a list of import discovery options that may be used
     User home directory and environment variables will be expanded.
 
 ``files`` (string)
+
     A comma-separated list of paths which should be checked by mypy if none are given on the command
-    line. Supports recursive file globbing using :doc:`library/glob`, where ``*`` (e.g. ``*.py``) matches
+    line. Supports recursive file globbing using :py:mod:`glob`, where ``*`` (e.g. ``*.py``) matches
```

Comment:
Please advise here. Both links lead to the same target: the [`glob`](https://docs.python.org/3/library/glob.html) module docs, but are rendered differently:

![image](https://user-images.githubusercontent.com/4455652/66438721-d6390600-ea2d-11e9-8c98-235b71166101.png)

vs

![image](https://user-images.githubusercontent.com/4455652/66438690-c28d9f80-ea2d-11e9-8e10-3de18cd92b22.png)

Reply:
I like the second better. So +1 to the change. (Do we use this `:doc:` style elsewhere, and does it get rendered so longwindedly there too? I think it's not needed.)

Reply:
There are a few `:doc:` roles that refer to standard library modules: 

https://github.com/python/mypy/blob/c95ecc1952b4f804662e37b36c0779b32226f88b/docs/source/additional_features.rst#L69

https://github.com/python/mypy/blob/c95ecc1952b4f804662e37b36c0779b32226f88b/docs/source/class_basics.rst#L235

https://github.com/python/mypy/blob/c95ecc1952b4f804662e37b36c0779b32226f88b/docs/source/config_file.rst#L32

https://github.com/python/mypy/blob/c95ecc1952b4f804662e37b36c0779b32226f88b/docs/source/error_code_list.rst#L545-L546

Do you suggest to replace them with the appropriate `:mod:` role?

Reply:
It doesn't seem necessary, since those other places all specify link text.

We cannot hope to strive for 100% consistency everywhere.